### PR TITLE
Snapshot mount to run on all FRM instances provided

### PR DESCRIPTION
**Pull request by priyankagupte15**
Results ;  http://192.168.1.62:8080/job/TrialJobs/job/PriyankaTrial/job/Ansible_Ubuntu20_Victoria-1_Func/28/HTML_20Report/
http://192.168.1.62:8080/job/TrialJobs/job/PriyankaTrial/job/Ansible_Ubuntu20_Victoria-1_Func/27/HTML_20Report/

http://192.168.1.62:8080/job/TrialJobs/job/PriyankaTrial/job/Ansible_Ubuntu20_Victoria-1_Func/30/HTML_20Report/

### CODE REVIEW

File: tempest/prerequisites.py [hunk 1]
```diff
@@ -918,7 +918,7 @@ def snapshot_mount(self):
         self.volumes_ids = []
         self.total_vms = 1
```
Comment (nasharatm) [inline tempest/prerequisites.py:919]: Please attach execution report for both centos and ubuntu frm? 
Please check "fvm_ssh_user = centos" field in tempest.conf

File: tempest/api/workloadmgr/snapshot/test_snapshot_mount.py [hunk 2]
```diff
@@ -32,235 +32,308 @@ def setup_clients(cls):
...
-                            "Verification of file's existance on moutned snapshot", tvaultconf.PASS)
+                            "Verify that mountpoint mounted is shown on FVM instance",
+                            tvaultconf.PASS)
+                        flag = 1
+                        if 'File_1' in i:
+                            LOG.debug("check that file exists on mounted snapshot")
+                            reporting.add_test_step(
+                                "Verification of file's existance on moutned snapshot", tvaultconf.PASS)
+                        else:
+                            LOG.debug("file does not found on FVM instacne")
```
Comment (nasharatm) [inline tempest/api/workloadmgr/snapshot/test_snapshot_mount.py]: Please check spell in few of LOG.debug statements.

File: tempest/api/workloadmgr/snapshot/test_snapshot_mount.py [hunk 3]
```diff
@@ -32,235 +32,308 @@ def setup_clients(cls):
...
+                reporting.test_case_to_write()
+
+            except Exception as e:
+                LOG.error("Exception: " + str(e))
+                reporting.set_test_script_status(tvaultconf.FAIL)
+                reporting.test_case_to_write()
+            index = index + 1
+
+    # @decorators.attr(type='workloadmgr_api')
+    # def test_2_umount_snapshot(self):
```
Comment (nasharatm) [inline tempest/api/workloadmgr/snapshot/test_snapshot_mount.py]: remove commented code.

**Review by nasharatm — CHANGES_REQUESTED** [review]

File: tempest/api/workloadmgr/snapshot/test_snapshot_mount.py (hunk 3 above)
Comment (priyankagupte15) [reply tempest/api/workloadmgr/snapshot/test_snapshot_mount.py]: done

File: tempest/prerequisites.py (hunk 1 above)
Comment (priyankagupte15) [reply tempest/prerequisites.py:919]: http://192.168.1.62:8080/job/TrialJobs/job/PriyankaTrial/job/Ansible_Ubuntu20_Victoria-1_Func/30/HTML_20Report/
updated in script

File: tempest/api/workloadmgr/snapshot/test_snapshot_mount.py (hunk 2 above)
Comment (priyankagupte15) [reply tempest/api/workloadmgr/snapshot/test_snapshot_mount.py]: updated

**Review by nasharatm — APPROVED** [review]

**Review by savitha-p — APPROVED** [review]

**Pull request merged**